contents/docs/data/identify.mdx

```diff
@@ -0,0 +1,561 @@
+// For User A
+posthog.identify('distinct_id_one');
+// For User B
+posthog.identify('distinct_id_two');
+
+// ❌ Not possible. 
+// You cannot use distinct_id_two as an alias for distinct_id_one
+client.alias({
+    distinctId: 'distinct_id_one',
+    alias: 'distinct_id_two',
+})
+```
+
+```python
+# For User A
+posthog.identify('distinct_id_one',{})
+# For User B
+posthog.identify('distinct_id_two',{})
+
+# ❌ Not possible
+# You cannot use distinct_id_two as an alias for distinct_id_one
+posthog.alias('distinct_id_two', 'distinct_id_one');
+```
+
+```php
+# For User A
+PostHog::identify(array(
+  'distinctId' => 'distinct_id_one',
+  'properties' => array()
+));
+# For User B
+PostHog::identify(array(
+  'distinctId' => 'distinct_id_two',
+  'properties' => array()
+));
+
+# ❌ Not possible
+# You cannot use distinct_id_two as an alias for distinct_id_one
+PostHog::alias(array(
+  'distinctId' => 'distinct_id_one',
+  'alias' => 'distinct_id_two'
+));
+```
+
+```ruby
+# For User A
+posthog.identify({
+  distinct_id: "distinct_id_one",
+  properties: {}
+})
+# For User B
+posthog.identify({
+  distinct_id: "distinct_id_two",
+  properties: {}
+})
+
+# ❌ Not possible
+# You cannot use distinct_id_two as an alias for distinct_id_one
+posthog.alias({
+  distinct_id: "distinct_id_one",
+  alias: "distinct_id_two",
+})
+```
+
+```go
+// For User A
+client.Enqueue(posthog.Identify{
+  DistinctId: "distinct_id_one",
+  Properties: posthog.NewProperties()
+})
+// For User B
+client.Enqueue(posthog.Identify{
+  DistinctId: "distinct_id_two",
+  Properties: posthog.NewProperties()
+})
+
+// ❌ Not possible
+// You cannot use distinct_id_two as an alias for distinct_id_one
+client.Enqueue(posthog.Alias{
+  DistinctId: "distinct_id_one",
+  Alias: "distinct_id_two",
+})
+```
+
+```java
+// For User A
+posthog.identify(
+  "distinct_id_one",
+  new HashMap<String, Object>()
+);
+// For User B
+posthog.identify(
+  "distinct_id_two",
+  new HashMap<String, Object>()
+);
+
+// ❌ Not possible
+// You cannot use distinct_id_two as an alias for distinct_id_one
+posthog.alias("distinct_id_one", "distinct_id_two");
+```
+
+</MultiLanguage>
+
+## Merging users
+
+It may happen that, due to implementation issues, the same user in your product has multiple users in PostHog associated with them. This occurs when, for the same user, you call `identify` with different distinct IDs. For example, if you used "email" as a distinct ID but then user changed their email. In these cases, you can use `merge` to merge multiple PostHog users into a single user.
+   
+> **Warning:** Merging users with `merge` is irreversible and has no safeguards! Be careful not to merge users who should not be merged together.
+> 
+> Due to the dangers, we don't recommend you merge users frequently, but rather as a one-off for recovering from implementation problems. We also encourage to consider whether merging users is strictly necessary for your data integrity, or a 'nice to have'.
```

Comment:
```suggestion
It may happen that, due to implementation issues, the same user in your product has multiple users in PostHog. After the implementation has been fixed to no longer create duplicates the special `$merge_dangerously` event can be used to merge any users in PostHog.
   
> **Warning:** `$merge_dangerously` has no safeguards! Misusage could result in all users being merged into one rendering any analytics useless. Use it only as one-off for recovering from implementation problems. 
```

we don't need an example here - what they will see is duplicate persons in PostHog. 
Move this below considerations - could be part of the fixing issues you saw from ingestion warnings 🤷‍♀️

Reply:
good idea